linters/scripts/find_space_indents.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -ex
+
+! rg --files-with-matches '^  ' "$(git rev-parse --show-toplevel)" | grep -vE ".json|.md|.LESSER|.eslintrc|.yaml|.txt"
```

Comment:
@Wayonb - not sure if we should integrate this into CI as we currently don't have place to run something across all files in repo and/or that ALWAYS runs 🤔

Reply:
Make sense.  Is the rg tool part of the new docker images?  If so we could add another step or as part of lint.